main/guides/dapps/README.md

```diff
@@ -24,6 +24,10 @@ If you have installed the [Agoric CLI](../getting-started/) and you want to try
 	cd my-fungible-faucet
 	# Install the project dependencies
 	agoric install
+	# Move to the contract directory
+ 	cd contract
+ 	# Run the yarn test command
+ 	yarn test
```

Comment:
That's perhaps worth adding on its own, but it would be better to actually document how to use our test framework. That is: it would be better to add a short version of https://github.com/Agoric/agoric-sdk/wiki/agoric-sdk-unit-testing